### PR TITLE
Feature/all planets view

### PR DESCRIPTION
**Pull request by alexthompson207**
# PR Rancid Tomatillos
**Connor A-L, Alex T & Katie B**

### What does this do?

- [ ] Fix
- [ X ] Feature
- [ ] Refactor
- [ ] Style
- [ ] Testing

### Are there any known bugs?

- [ X ] No
- [ ] Yes (if so please disclose in Notes for Reviewer & Next Iterations)

### Summary of the changes
Add a Planet and Planetarium component with tsx and css files. Create basic planet card structure from test data to display planet image, planet name, and planet fact on card. Also create basic css styling for Planetarium and Planet component.

### What ticket(s) do the changes resolve?

#11 

Please Link Issues to this PR as necessary.

### Notes for the Reviewer

This is only using test data for now. Feel free to adjust interface and className names.

### How to Test Changes?

Pull down this branch, run `npm i` then `npm start`. All 4 test data planets should appear on the DOM.

### Next Iterations

Create an image folder holding all planet images, link image maybe by id to planet card. Create a function to display planet fact, i.e. `Distance from the Sun` or `Average Temperature` -- needs to be dynamic.

### CODE REVIEW

File: src/Planet/planet-images/earth.svg [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+</svg>
```
Comment (knees4bees) [inline src/Planet/planet-images/earth.svg:89]: Noting for our future selves that this is a placeholder that can be safely deleted later.

File: src/Planet/Planet.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import earth from './planet-images/earth.svg'
+
+interface PlanetData {
+  name: string,
+  id: number,
+  planetFact: number
+}
+
+
+const Planet: React.FC<PlanetData> = ({ id, name, planetFact }): JSX.Element => {
```
Comment (knees4bees) [inline src/Planet/Planet.tsx:12]: Nice job with the TS typing here

**Review by ConnorAndersonLarson — APPROVED** [review]
Looks good, we can swap out the svg later. Great work!

**Review by knees4bees — APPROVED** [review]
This is great work, Alex! I'm ready to merge if @ConnorAndersonLarson approves.

**Pull request merged**